### DISCUSSION TITLE
Type error: Argument of type 'ChainProviderFn<JsonRpcProvider, WebSocketProvider, Chain>[]'

### DISCUSSION THREAD

**Discussion by ranjithpitsync** [wevm/wagmi#1521] [General]
When I tried to **yarn run build** I get this error

> Type error: Argument of type 'ChainProviderFn<JsonRpcProvider, WebSocketProvider, Chain>[]' is not assignable to parameter of type 'ConfigureChainsConfig | undefined'.
> 
>   88 |     publicProvider(),
>   89 |   ],
> > 90 |   [
>      |   ^
>   91 |     jsonRpcProvider({
>   92 |       rpc: (chain) => {
>   93 |         if (chain.id !== binanceChain.id) return null;

**Comment by afa7789** [comment]
how you fixed it ?
